index.bs

```diff
@@ -2946,6 +2946,135 @@ and makes it clear when the state described by the flags is reset.
 
 <!-- TODO: add examples -->
 
+<h3 id="implementability">Resolving tension between interoperability and implementability</h3>
+
+<!--
+See "How do we balance concerns between interoperability and implementability?"
+https://github.com/w3ctag/design-principles/issues/142
+  -->
+
+Specifying a feature,
+like all engineering work,
+requires weighing tradeoffs
+and compromising.
+
+Sometimes,
+despite the best efforts of all involved,
+we fail to find a way
+to interoperably specify a feature
+that every implementor agrees
+is implementable in their engine.
+
+Choosing a way forward should be guided by
+[what is best for the end user](#priority-of-constituencies).
+
+First,
+examine the nature of the implementability concerns.
+Perhaps the implementor identified potential end-user harm
+and decided that the feature should not be implemented.
+In this situation,
+it may be best to not specify the feature,
+and for any implementors who have shipped the feature
+to un-ship it.
+Un-shipping features may result in user and author
+confusion.
+
+<div class=example id=sharedarraybuffer>
+
+When the Spectre attack was discovered,
+{{SharedArrayBuffer}} was unshipped in every implementation
+because it can be used to create a high-precision timer
+that could be used for a Spectre attack.
+
+</div>
+
+Sometimes the same API is implementable in every engine,
+but its behavior cannot be made to be fully interoperable.
+Whenever behavior differs between implementations,
+the end-user impact must be considered.
+One downside to this approach
+is that the difference in behavior is not [feature detectable](#feature-detect).
+On the other hand,
+if something changes in the future
+that enables all implementations to converge on the same behavior,
+sites will not need to be updated
+to take advantage of this.
+
+Note: Authors may assume the behavior of the dominant implementation is correct
+and any other behavior is buggy,
+which may further entrench the dominant implementaiton.
+
+<div class=example id=unknown1>
+
+The 'backdrop-filter' property is known to have a number of visible
+behavior differences between implementations.
+At the present time,
+those working on it believe
+the benefits of sharing an API
+outweigh the interoperability costs of the differences.
+
+</div>
+
+If the behavior differences are serious enough,
+it may be preferable to specify different APIs.
+
+In this scenario,
+each implementor goes with the alternative
+that they are able to implement.
+If implementations are eventually able to converge on the same behavior,
+both APIs may need to be supported indefinitely,
+which can increase implementation maintenance costs.
+
+Exposing two different APIs
+that are intended to address the same use cases
+can be confusing to authors.
+Authors can use [feature detection](#feature-detect)
```

Comment:
```suggestion
forces authors to use [feature detection](#feature-detect)
```

Reply:
this suggestion doesn't appear to be grammatical.

Reply:
As discussed today we think this is OK.